extensions/ql-vscode/src/types/query-language.ts

```diff
@@ -0,0 +1,37 @@
+export enum QueryLanguage {
```

Comment:
As discussed separately, we could put this somewhere (in `common`?) instead of creating a new folder 📁 💥